RELEASING.md

```diff
@@ -2,6 +2,9 @@
 
 _Instruction for Maintainers only._
 
+- Run [opentelemetry.io workflow](https://github.com/open-telemetry/opentelemetry.io/actions/workflows/build-dev.yml)
```

Comment:
How do you run this against a submodule? Can you please provide more details. I don't know how to do this.

Reply:
1. open that link
2. type opentelemetry-proto into the submodule field 
<img width="1414" alt="image" src="https://github.com/user-attachments/assets/9af9ba45-19a0-4ab4-a086-b01245ac9979" />
3. Run it
4. wait for it to complete
6. Both `Build and check links` step along with `warnings in build log?` should pass.
7. If any of them fails, check  `Build and check links`/`Run npm run log:test-and-fix` logs - broken links should be there
8. Ignore "REFCACHE updates?" result. 

Did I capture it right @chalin ?

run with broken links

<img width="1598" alt="image" src="https://github.com/user-attachments/assets/f782a302-b539-4db9-a6e0-a8571940d7af" />

run without issues:

<img width="1356" alt="image" src="https://github.com/user-attachments/assets/e060b2c1-f071-4a7e-8e94-c10f041ae564" />

I don't think we normally go into this level of details in release doc, but it's easy to figure out the moment you try it.

Reply:
Yes, thanks about it. 👍🏻

Reply:
Great, thanks! Would it be possible to add the distilled version of this to RELEASING.md?